nautobot/docs/user-guide/administration/installation/nautobot.md

```diff
@@ -94,30 +94,29 @@ Example output:
 
 Because the `nautobot` user was created with `NAUTOBOT_ROOT` set as its home directory and we had you set the shell to `/bin/bash`, the binary path `$NAUTOBOT_ROOT/bin` is automatically added to the beginning of the `$PATH` environment variable:
 
-In Ubuntu 20.04:
+=== "Ubuntu 20.04"
```

Comment:
What's this `===` syntax? I haven't seen that before in Markdown...

Reply:
This is what enables the tabbed interface within the mkdocs theme. So each `=== "Section Title"` is the section title. Everything indented at that point falls under the section until either another `=== "Section 2 Title"` is found, or it is de-indented, which then is back outside of the tab.

Reply:
An example with it is here of the output - https://docs.nautobot.com/projects/core/en/stable/installation/http-server/#obtain-an-ssl-certificate